opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.cpp

```diff
@@ -0,0 +1,61 @@
+#include "OilVaporizationProperties.hpp"
+
+namespace Opm {
+
+
+    OilVaporizationProperties::OilVaporizationProperties(){
+
+    }
+
```

Comment:
The different `getXXX()` methods should be protected with a check of the type variable - i.e. asking for `m_maxDRSDT` when `m_type == VAPPARS` is an internal error and should give an exception.